src/components/Connect.vue

```diff
@@ -24,16 +24,16 @@ watch(
   value => (hasExtension.value = value)
 )
 
-const connectWalletConnect = async () => {
+const connectWalletConnectV2 = async () => {
   close(dropdown.value)
-  await setupProvider()
-  await enableProvider()
+  await setupWCV2Provider()
+  await openWCV2Modal()
```

Comment:
For some reason it doesn't trigger a modal window with the QR code to connect to DApp.
Will look into this later today.